src/tokenizer/index.ts

```diff
@@ -97,7 +96,7 @@ export function tokenize(
 }
 
 function trim(text: string[]): string[] {
-  while (text[text.length - 1] === "") {
+  while (text.at(-1) === "") {
```

Comment:
`String.prototype.at(-1)` is slow.

Reply:
https://github.com/RafaelGSS/nodejs-bench-operations/blob/main/RESULTS-v18.md#get-the-last-item-of-an-array

Reply:
Yeah I guess direct pointer access will always be faster

Reply:
I didn't knew this. Thanks for sharing the benchmark, I'll use it as a bible 😄